### PR TITLE
Update ssm middleware to accept paths

### PR DESCRIPTION
**Pull request by garyhollandxyz**
Introduces functionality discussed in #142. I thought a working example might help!

With this version of the ssm middleware, you can specify both a `names` object and a `paths` object in `config`. (I.e. a map of `LOCAL_NAME -> PARAMETER_NAME` and `LOCAL_NAME -> PARAMETER_PATH` respectively.

This is currently closer to @lmammino's suggested approach.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,521 +1,521 @@
...
+ - [`httpContentNegotiation`](/docs/middlewares.md#httpcontentnegotiation): Parses `Accept-*` headers and provides utilities for content negotiation (charset, encoding, language and media type) for HTTP requests
+ - [`httpErrorHandler`](/docs/middlewares.md#httperrorhandler): Creates a proper HTTP response for errors that are created with the [http-errors](https://www.npmjs.com/package/http-errors) module and represents proper HTTP errors.
+ - [`httpEventNormalizer`](/docs/middlewares.md#httpEventNormalizer): Normalizes HTTP events by adding an empty object for `queryStringParameters` and `pathParameters` if they are missing.
+ - [`httpHeaderNormalizer`](/docs/middlewares.md#httpheadernormalizer): Normalizes HTTP header names to their canonical format
+ - [`httpPartialResponse`](/docs/middlewares.md#httppartialresponse): Filter response objects attributes based on query string parameters.
+ - [`jsonBodyParser`](/docs/middlewares.md#jsonbodyparser): automatically parses HTTP requests with JSON body and converts the body into an object. Also handles gracefully broken JSON if used in combination of
```
Comment (dkatavic) [inline README.md]: Is there a reason that the `automatically` is lowercased?

Comment (garyhollandxyz) [reply README.md]: Definitely not one that I know of! I'll update it.

**Comment by garyhollandxyz** [comment]
@lmammino thanks for the response! That all sounds good to me. I'll update the PR as soon as I'm able.

**Comment by garyhollandxyz** [comment]
@lmammino amazing, thanks!

**Pull request merged**